storage/local/persistence.go

```diff
@@ -718,6 +715,7 @@ func (p *persistence) loadSeriesMapAndHeads() (sm *seriesMap, err error) {
 					return sm, nil
 				}
 				chunkDescs[i] = newChunkDesc(chunk)
+				chunkDescsTotal-- // Avoid double-counting by newChunkDesc.
```

Comment:
Hmm, do we care that we have a bunch of nil-error return cases above which would cause a too high count (like dirtiness and not being able to decode a chunk type)?